### PR TITLE
cleaned up line tactic for use in penalty plays

### PR DESCRIPTION
**Pull request by sseye6**
## Description
Cleaned up line tactic so that the robots start on the side of the field for penalty kicks.  

### Test Case 1
1. run line_up play on sim

Expected result: The robots should start on the side of the field

### CODE REVIEW

File: rj_gameplay/rj_gameplay/tactic/line_tactic.py [hunk 1]
```diff
@@ -22,7 +30,7 @@ def __init__(self, world_state: stp.rc.WorldState):
         for pt in self.move_points:
             # for some reason stp.role doesn't need to be imported?
             self._role_requests.append(
-                (stp.role.cost.PickClosestToPoint(pt), dumb_move.DumbMove)
+                (stp.role.cost.PickClosestRobot(pt), dumb_move.DumbMove)
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/line_tactic.py]: hey, I think the code changed under you while you were working, but the costFn's name is now `PickClosestToPoint` (see `stp/role/cost.py`)

File: rj_gameplay/rj_gameplay/tactic/line_tactic.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import stp
 from rj_gameplay.role import dumb_move
 from typing import List, Tuple
-
+from stp.rc import (
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/line_tactic.py]: fix these style errors by removing unused imports

**Review by kfu02 — CHANGES_REQUESTED** [review]
Hey, code looks good, but there's a few things to clean up. (See review comments)

File: rj_gameplay/rj_gameplay/tactic/line_tactic.py [hunk 3]
```diff
@@ -13,7 +12,11 @@ def __init__(self, world_state: stp.rc.WorldState):
 
         # compute move points
         # TODO: make start on side of the field, so this Tactic is actually useful during penalty situations
-        start = (3.0, 0.0)
+        # access the length and the width of the field (minus border), and starts robots there
+        start = (
+            (world_state.field.length_m - world_state.field.border_m) / 2,
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/line_tactic.py:17]: these seem to be the wrong coordinates, do you get the correct behavior on your end when you run sim?

**Review by kfu02 — CHANGES_REQUESTED** [review]
The line up play actually launches now, but it shoves all the robots off of the field, have you tested this locally?

**Review by HussainGynai — APPROVED** [review]
LTGM

File: rj_gameplay/rj_gameplay/tactic/line_tactic.py [hunk 4]
```diff
@@ -13,10 +12,14 @@ def __init__(self, world_state: stp.rc.WorldState):
 
         # compute move points
         # TODO: make start on side of the field, so this Tactic is actually useful during penalty situations
-        start = (3.0, 0.0)
+        # access the length and the width of the field (minus border), and starts robots there
+        start = (
+            (world_state.field.width_m - world_state.field.border_m) / 2,
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/line_tactic.py]: can you switch this minus to a plus so the robots stand slightly outside of the field lines?

File: rj_gameplay/rj_gameplay/tactic/line_tactic.py [hunk 5]
```diff
@@ -13,10 +12,14 @@ def __init__(self, world_state: stp.rc.WorldState):
 
         # compute move points
         # TODO: make start on side of the field, so this Tactic is actually useful during penalty situations
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/line_tactic.py]: you can delete this TODO also

**Review by kfu02 — CHANGES_REQUESTED** [review]
some light changes

**Review by CK-Christopher — APPROVED** [review]
Everything looks good to me!

**Pull request merged**